### DISCUSSION TITLE
There is a new controller version available, when will you update?!

### DISCUSSION THREAD

**Discussion by dereitz** [hassio-addons/addon-unifi#150] [General]
# Problem/Motivation

When I login to the controller, I receive a popup that a new version is available: 6.0.45.

## Proposed changes

Update of add-on to include new version of controller.

**Comment by frenck** [comment]
I roll updates regularly. However, I do not blindly push an update without ensuring it works.

The add-on is used by thousands of systems. It would be irresponsible to do so without testing.

Opening issues for that has no use.

**Comment by dereitz** [comment]
No disagreement at all on your position.  I wasn't sure if you were aware of the new version or not.  Sounds like you're well aware.  How do you prefer to receive updates about new versions or do you already have that covered?

**Reply by frenck** [reply]
In this case, I've received a PR and that has been merged already.
The update is in the current development version.

**Reply by frenck** [reply]
v0.20.1 is building and released. Should be available on your system any moment now.
